processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -355,6 +357,9 @@ public int lookupId(String name)
   private final Map<String, ColumnCapabilitiesImpl> columnCapabilities;
   private final List<DimDim> dimValues;
 
+  // looks need a configuration
+  private final Ordering<Comparable> ordering = Ordering.natural().nullsFirst();
```

Comment:
@jon-wei I'm not sure I understand what the comment means, can we have a test for this?